.azure/OneBranch.Package.yml

```diff
@@ -23,111 +32,101 @@ parameters:
-  - push_amd64fre
-  - push_amd64chk
-  - push_arm64fre
-  - push_arm64chk
-  - push_armfre
-  - push_armchk
-  - push_chpefre
-  - push_chpechk
-  - push_x86fre
-  - push_x86chk
-  jobs:
-  - template: ./obtemplates/git-submit.yml
+  ROOT: $(Build.SourcesDirectory)
+  REPOROOT: $(Build.SourcesDirectory)
+  OUTPUTROOT: $(REPOROOT)\out
+  NUGET_XMLDOC_MODE: none
+  WindowsContainerImage: 'onebranch.azurecr.io/windows/ltsc2022/vse2022:latest'
+
+extends:
+  template: v2/Microsoft.Official.yml@templates # https://aka.ms/obpipelines/templates
+  parameters:
+    platform:
+      name: 'windows_undocked' # windows undocked
+    cloudvault: # https://aka.ms/obpipelines/cloudvault
+      enabled: false
+    globalSdl: # https://aka.ms/obpipelines/sdl
+      tsa:
+        enabled: false # onebranch publish all sdl results to TSA. If TSA is disabled all SDL tools will forced into 'break' build mode.
+      credscan:
+        suppressionsFile: $(Build.SourcesDirectory)\.azure\CredScanSuppressions.json
+      binskim:
+        break: true # always break the build on binskim issues in addition to TSA upload
+      policheck:
+        break: true # always break the build on policheck issues. You can disable it by setting to 'false'
+      suppression:
+        suppressionFile: $(Build.SourcesDirectory)\.azure\openssl.gdnsuppress
+    featureFlags:
+      WindowsHostVersion: '1ESWindows2022'
+
+    stages:
+    - stage: prepare
+      pool:
+        type: windows
+      displayName: Prepare VPack
+      dependsOn: []
+      jobs:
+      - template: .azure/obtemplates/prepare-vpacks.yml@self
+
+    - stage: build_vpack
+      displayName: Build VPack
+      dependsOn: [prepare]
+      jobs:
+      - job: package
+        pool:
+          type: windows
+        strategy:
+          matrix:
+            amd64fre:
+              platform: amd64fre
+            amd64chk:
+              platform: amd64chk
+            arm64fre:
+              platform: arm64fre
+            arm64chk:
+              platform: arm64chk
+            armfre:
+              platform: armfre
+            armchk:
+              platform: armchk
+            chpefre:
+              platform: chpefre
+            chpechk:
+              platform: chpechk
+            x86fre:
+              platform: x86fre
+            x86chk:
+              platform: x86chk
+        variables:
+          runCodesignValidationInjection: false
+          group: VPackDeploymentKeys
+          ob_outputDirectory: $(Build.SourcesDirectory)/artifacts/package/$(platform)
+          ob_artifactSuffix: _$(platform)
+          ob_sdl_codeSignValidation_excludes: -|**\*.sys;-|**\*.dll;-|**\*.exe
+          ob_createvpack_enabled: true
+          ob_createvpack_packagename: msquic.$(platform)
+          ob_createvpack_owneralias: quicdev
+          ob_createvpack_description: msquic.$(Build.SourceBranchName)
+          ob_createvpack_versionAs: string
+          ob_createvpack_version: 2.3.0-$(Build.BuildId)
```

Comment:
Does our update version ps1 script still work for this, or does it need to be updated?

Reply:
Actually, there is an issue. The BuildId in vpack version won't match  MSQUIC_METADATA_ONEBRANCH_BUILD_ID.

Reply:
MSQUIC_METADATA_ONEBRANCH_BUILD_ID won't match the build ID of the pipeline that pushes vpack and creates the PR.

Reply:
Looks like we have been doing all the time. So, probably not a big deal.

Reply:
>> Does our update version ps1 script still work for this, or does it need to be updated?
nvm. I updated update-version.ps1.